boa/src/exec/tests.rs

```diff
@@ -967,7 +966,7 @@ fn to_length() {
     assert_eq!(
         Value::number(100000000000.0)
             .to_length(&mut engine)
-            .unwrap(),
+            .unwrap() as u128,
```

Comment:
we return a `usize` from `to_length` so we are already casting as `usize` so this does not do anything.

Reply:
we should return a `u64` instead of `usize` since its clamped at 0 .. `2**53 - 1`

Reply:
Changed :)